packages/cactus-plugin-ledger-connector-go-ethereum-socketio/src/main/typescript/common/core/bin/www.ts

```diff
@@ -89,7 +89,7 @@ function normalizePort(val) {
  * Event listener for HTTPS server "error" event.
  */
 
-function onError(error) {
+function onError(error: any) {
```

Comment:
@outSH A recommendation for the future: change the `any` to `unknown` so that the compiler forces proper runtime type checking on the developers during development (it will force them to add checks to prevent this function crashing if `error` is null or undefined for example). Not a change request, just something to keep in mind for the future as a recommendation.
Also apologies for the slow reviews, I had a lot of trouble with reviving the CI but we are back in business now.